q2_sample_classifier/_transformer.py

```diff
@@ -65,13 +65,14 @@ def _4(data: pd.Series) -> (PredictionsFormat):
 def _5(ff: PredictionsFormat) -> (pd.Series):
     with ff.open() as fh:
         df = _read_dataframe(fh)
-        return df.iloc[:, 0]
+        return pd.to_numeric(df.iloc[:, 0], errors='ignore')
```

Comment:
this change is the meat of this PR. previously importances that were saved in scientific notation were not being cast correctly, see #127 . This issue could also theoretically impact numeric predictions and probabilities.

I did not want to touch the general behavior of `_read_dataframe` (which disables casting on import), so instead cast to numeric on return. The `errors='ignore'` part is so that we only cast numeric columns. E.g., `PredictionsFormat` could contain numeric or str values. Importances should be all numeric so I suppose I should change that transformer to `errors=True`?

Reply:
Cool, makes sense to me. Is there a test or two we can throw in for this?

Reply:
yep. we already have a test for numeric values (note how in one of the tests in the changelog floats were previously being cast as strings! 😆 ) and I am about to push a new test for categorical predictions.